framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -82,7 +82,6 @@ qx.Mixin.define("qx.ui.core.MExecutable",
      */
     _bindableProperties :
     [
-      "enabled",
```

Comment:
Well it doesn't sound to bad if I have a command which is attached to multiple buttons, is able to set all attached buttons to "disabled" via one command.setEnabled(false). 

I've no example for multiple buttons, but the demo browser example http://www.qooxdoo.org/current/demobrowser/#widget~Menu.html does exactly that, it attaches a command CTRL+V to the paste button and disables the command which also disables the paste button. See the source here:
https://github.com/qooxdoo/qooxdoo/blob/master/application/demobrowser/source/class/demobrowser/demo/widget/Menu.js#L133

Similar code would definitely break!